### DISCUSSION TITLE
Graphql many to many example

### DISCUSSION THREAD

**Discussion by roei3000b** [prisma/prisma#11494] [Q&A] [answered]
Hey, I'm using prisma and graphql.
I want to add many to many connection between two types and the connection between them should also have information that describes the connection (time of when the connection was added, who added the connection etc...)
In prisma I did it with connection table, how do I add this connection table in correct way to graphql?

**Comment by nikolasburk** [comment, marked as answer]
Hey @roei3000b 👋  can you share your Prisma models (incl the relation table) and what library you're using to create the GraphQL schema (e.g. Nexus, TypeGraphQL or SDL-first via e.g. via Apollo Server)? 

Generally, the approach to take here is similar to any other relation that you implement via GraphQL and you'll need to implement _type resolvers_ for this. I would guess though that in this case you _probably_ don't want to expose the relation table via GraphQL, so there is a question about how to expose the data from that relation table if it's supposed to be queryable from the frontend 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
